docs/release-notes/2.34.0.md

```diff
@@ -0,0 +1,107 @@
+
+# Percona Monitoring and Management 2.34.0
+
+
+| **Release date:** | [January 24, 2022]                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+
+
+## Release Highlights
+
+### UX enhancements to Backup Management
+
+With this release, we are updating the Backup and Restore functionality with key usability enhancements, including:
+
+- Filtering on services at backup creation time. This makes it considerably easier to find the right service for a backup when working with multiple servers.
+- Automatic redirection to the **Restores** page after initiating a restore. From here you can immediately monitor the restore process, and see an overview of all the restore jobs for your PMM instance.
+- Added information about the storage type for the backups listed on **All backups** page. This helps you get a more comprehensive overview of your current backups.
+- Design and usability improvements on the **Create Backup** and **All Backups** page to increase consistency with other PMM UI controls.
+
+### Components Upgrade
+- List the components which are upgraded as part of the Release.
+
+## New Features
+
+    
+
+- [PMM-11031](https://jira.percona.com/browse/PMM-11031) - By using the Expose field, DBaaS users can easily connect a database cluster in a secure environment. You can modify the exposure level using the Source Range field.
+- [PMM-11086](https://jira.percona.com/browse/PMM-11086) - Improved design for the **On Demand** and **Schedule Backup** cards on the **Create Backup** page. In addition, the **Created** column on the **Restores** tab has been replaced with **Started at** and **Ended** columns, to provide more insight into the restore process. 
+
+
+## Improvements
+
+- [PMM-11024](https://jira.percona.com/browse/PMM-11024) - [DBaaS]: Move from dbaas-controller to dbaas-operator
```

Comment:
@gen1us2k What is the user impact for this ticket?

Reply:
I'll add notes